tensorflow/core/common_runtime/mkl_layout_pass_test.cc

```diff
@@ -5166,7 +5159,6 @@ static void BM_MklLayoutRewritePass(int iters, int op_nodes) {
     }
     iters -= N;  // Our benchmark units are individual graph nodes,
                  // not whole graphs
-    // delete graph;
```

Comment:
Shouldn't this be uncommented instead?

Reply:
yes, I will do

Reply:
@penpornk  I double-checked and found that this is a commented out statement, which should be deleted.

Reply:
Thank you for double-checking! But won't we have memory leaks without this line (`graph` gets reassigned to a new address in line 5154 in each iteration). 

Maybe change line 5154 to 
```c++
std::unique_ptr<Graph> graph(new Graph(OpRegistry::Global()));
``` 
instead?

Reply:
Good suggestion, I will take a look. 

The current implementation may have mem leaky issue, which has not been detected due to being test only :)

Reply:
Done with code change (per suggestion of using unique_ptr) and with successful local test!